### PR TITLE
Add R4 Encounter sorting details, update Pagination with previous link

### PR DESCRIPTION
**Pull request by FFSol**
Description
----

- Document that R4 Encounter Bundles are sorted by the Encounter.period
- Update Bundle Page links to include the Previous link

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by FFSol** [comment]
Changes:
----
Updated Encounter Implementation Details
Screen Shot 2020-05-12 at 2 28 07 PM
----
Added Previous link
Screen Shot 2020-05-12 at 2 32 12 PM
----

File: content/millennium/r4/encounters/encounter.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ _Implementation Notes_
 
 * The [Encounter.hospitalization.destination] will be returned as a reference to a [contained] location resource.
 * The [Encounter.location.location] may be returned as a reference to a [contained] location resource.
+* A populated Encounter response bundle will be sorted based on the [Encounter.period].
```
Comment (daliboz) [inline content/millennium/r4/encounters/encounter.md:59]: I think it may be specifically sorted on the start/beginning? If so, call that out.

Comment (FFSol) [reply content/millennium/r4/encounters/encounter.md]: https://github.com/cerner/fhir.cerner.com/pull/381/commits/4d18a64cdcda5172c7387f7486acfbcf94b53713

File: content/millennium/dstu2.md [hunk 2]
```diff
@@ -533,6 +533,7 @@ The possible `relation` values are:
 --------|------------------------------------------------------
  `self` | Shows the URL of the current page of results.
  `next` | Shows the URL of the immediate next page of results.
```
Comment (beichhor) [inline content/millennium/dstu2.md:535]: nit: can we keep the table columns aligned in the md?

Comment (FFSol) [reply content/millennium/dstu2.md]: https://github.com/cerner/fhir.cerner.com/pull/381/commits/97145ae812d544d0be308122b426766d2b00c5cd

File: content/millennium/r4/encounters/encounter.md [hunk 3]
```diff
@@ -56,6 +56,7 @@ _Implementation Notes_
 
 * The [Encounter.hospitalization.destination] will be returned as a reference to a [contained] location resource.
 * The [Encounter.location.location] may be returned as a reference to a [contained] location resource.
+* A populated Encounter response bundle will be sorted based on the start of the [Encounter.period].
```
Comment (beichhor) [inline content/millennium/r4/encounters/encounter.md:59]: Could we maybe clarify this a little bit more? Is it oldest -> newest or newest -> oldest?

Comment (FFSol) [reply content/millennium/r4/encounters/encounter.md]: https://github.com/cerner/fhir.cerner.com/pull/381/commits/bfe93e45d4e2ea096da8bf9e80782f015c54bdc1

Comment (FFSol) [reply content/millennium/r4/encounters/encounter.md]: Screen Shot 2020-06-12 at 8 27 27 AM

**Comment by ebaranowski** [comment]
Screen Shot 2020-06-23 at 5 02 13 PM

Validated

**Pull request merged**